### DISCUSSION TITLE
External input (bluetooth controller) inputs while navigating

### DISCUSSION THREAD

**Discussion by crevatsch** [osmandapp/OsmAnd#20118] [Q&A]
Hi everyone! I have a problem with an external input device (the JaxeADV Barbuttons) in Osmand. When i open the app without a route planned and no widgets on the screen, the barbuttons work perfectly (panning, zoom, etc). When i plan a route however the buttons stop working and only want to select the widgets on the screen (like speed, next turn, 3d or 2d mode, etc). Is there a way to retain the ability to control the map and not the widgets while navigating?

**Comment by Sonwon1** [comment]
I upvoted this, keeping my hands on the handlebars is a safety issue.

**Comment by vshcherb** [comment]
If you switch navigation mode it could be that keyboard settings are not set cause they are profile dependent.

**Comment by crevatsch** [comment]
Thanks for the suggestion. I only have one profile active (browse map) to reduce errors between different profile settings. External input devices is set to ''keyboard'' in this profile.
